testdata/wait-epmd.ps1

```diff
@@ -0,0 +1,25 @@
+$running = $false
+[int]$count = 1
+
+if ($env:ERLANG_HOME) {
+    $epmd = Join-Path -Path $env:ERLANG_HOME 'erts-8.3\bin\epmd.exe'
+} else {
+    $epmd = 'C:\Users\appveyor\erlang\erts-8.3\bin\epmd.exe'
```

Comment:
Why wouldn't ERLANG_HOME be set? If it isn't, it would seem to be an error in our AppVeyor script, and we are better off fixing that rather than hardcoding the location in multiple places. If ERLANG_HOME isn't set up, let's cause the build to fail with a helpful error message.